tools/agenda-generator/src/generator.rs

```diff
@@ -497,6 +508,23 @@ impl GithubQuery {
 
                 let issues = github_api(&endpoint)?;
                 let issues = generator.dedup(issues);
+
+                let excluded_labels: BTreeSet<_> = self
+                    .excluded_labels
+                    .iter()
+                    .flat_map(|labels| labels.iter())
+                    .map(|s| s.to_string())
+                    .collect();
+                let issues = issues.filter(|issue| {
+                    for excluded_label in &excluded_labels {
+                        if issue.labels.contains(&excluded_label) {
+                            return false;
+                        }
+                    }
+
+                    true
+                });
```

Comment:
With this, `.exclude_labels(&["a", "b"])` and `.exclude_labels(&["a"]).exclude_labels(&["b"])` will be identical, right?

I liked what you did with `.labels(&["a", "b"])`, which requires both labels to be present. I suppose `.exclude_labels(&["a", "b"])` could do the same and exclude issues that have both `a` and `b`, to allow for more flexibility:

```suggestion
                let issues = issues.filter(|issue| {
                    !self.excluded_labels.iter().any(|labels| {
                        labels
                            .iter()
                            .all(|&label| issue.labels.iter().any(|x| x == label))
                    })
                });
```